sklearn/linear_model/ridge.py

```diff
@@ -251,18 +251,22 @@ def ridge_regression(X, y, alpha, sample_weight=None, solver='auto',
           scipy.sparse.linalg.lsqr. It is the fastest but may not be available
           in old scipy versions. It also uses an iterative procedure.
 
-        - 'sag' uses a Stochastic Average Gradient descent. It also uses an
-          iterative procedure, and is often faster than other solvers when
-          both n_samples and n_features are large. Note that 'sag' fast
-          convergence is only guaranteed on features with approximately the
-          same scale. You can preprocess the data with a scaler from
-          sklearn.preprocessing.
+        - 'sag' uses a Stochastic Average Gradient descent, and 'saga' uses
+          its improved, unbiased version named SAGA. Both methods also use an
```

Comment:
"improved" is subjective and Nicolas Le Roux won't be our friend anymore ;) Let's stick to objective qualifiers such as "unbiased" and "more flexible" (because of support for non-smooth penalty).